code/lib/theming/src/tests/util.test.js

```diff
@@ -100,9 +100,12 @@ describe('utils', () => {
       // By setting matches to always be true any checks for prefer-color-scheme
       // will match and since we only check if the preferred scheme is dark this
       // is a simple way to test it
-      globalWindow.matchMedia = vi.fn().mockImplementation(() => ({
-        matches: true,
-      }));
+      vi.stubGlobal(
+        'matchMedia',
+        vi.fn().mockReturnValue({
+          matches: true,
+        })
+      );
```

Comment:
We can also just use `globalThis.matchMedia = ` directly, but then we can't use `vi.unstubAllGlobals` if we need it.  (ref https://vitest.dev/api/vi.html#vi-stubglobal)